input/postgres/relation_stats_ext.go

```diff
@@ -0,0 +1,105 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/lib/pq"
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+)
+
+const extendedStatisticsSQLExprsField = "ARRAY[]::text[]"
+const extendedStatisticsSQLpg14ExprsField = "se ->> 'exprs'"
```

Comment:
[note for reviewer] Support for defining extended statistics on expressions started from Postgres 14, so it is okay to simply give an empty text array below Postgres 14.